sqladmin/models.py

```diff
@@ -288,6 +304,40 @@ class UserAdmin(ModelAdmin, model=User):
     edit_template: ClassVar[str] = "edit.html"
     """Edit view template. Default is `edit.html`."""
 
+    # Export
+    column_export_list: ClassVar[List[Union[str, InstrumentedAttribute]]] = []
+    """List of columns to include when exporting.
+    Columns can either be string names or SQLAlchemy columns.
+
+    ???+ example
+        ```python
+        class UserAdmin(ModelAdmin, model=User):
+            column_export_list = [User.id, User.name]
+        ```
+    """
+
+    column_export_exclude_list: ClassVar[List[Union[str, InstrumentedAttribute]]] = []
+    """List of columns to exclude when exporting.
+    Columns can either be string names or SQLAlchemy columns.
+
+    ???+ example
+        ```python
+        class UserAdmin(ModelAdmin, model=User):
+            column_export_exclude_list = [User.id, User.name]
+        ```
+    """
+
+    export_types: ClassVar[List[str]] = ["csv"]
+    """A list of available export filetypes.
+    Currently only `csv` is supported.
+    """
+
+    export_max_rows: ClassVar[Optional[int]] = 0
+    """Maximum number of rows allowed for export.
+
+    Unlimited by default. Uses `page_size` if set to `None`.
+    """
```

Comment:
What do you think if we don't follow the Flask-Admin behaviour here? I don't think it would be a problem here.

```suggestion
    export_max_rows: ClassVar[int] = 0
    """Maximum number of rows allowed for export.
    Unlimited by default.
    """
```

Reply:
I think Flask-Admin's default behavior is very weird-- It's odd that `0` is default and means infinite, and `None` is a choice and has a completely different behavior. I think deviating is justifiable.